yt_dlp/extractor/nhk.py

```diff
@@ -539,9 +539,31 @@ class NhkRadiruIE(InfoExtractor):
         },
     }]
 
+    _api_config = None
+
+    def _extract_extended_description(self, episode_id, aa_vinfo2, aa_vinfo3):
+        service, area = aa_vinfo2.split(",")
+
+        if not self._api_config:
```

Comment:
checking truthiness of an etree element is no longer supported in py3.12 so we have to do this bit of cringe
```suggestion
        if not isinstance(self._api_config, xml.etree.ElementTree.Element):
```
&nbsp;
and then above
```py
import xml.etree.ElementTree
```